supporting_scripts/course-scripts/quick-course-setup/README.md

```diff
@@ -0,0 +1,133 @@
+# Artemis Course Setup Scripts
+
+This project contains Python scripts that automate the setup and management of courses and users in an Artemis instance. The scripts help you create a course, generate users, enroll them into courses, and manage programming exercises, including participation and commits. The project is useful for setting up test environments or quickly initializing a course with a large number of students.
+
+## Setup
+
+### 1. Install Python and the Python Plugin for IntelliJ
+
+- Ensure that Python (preferably version 3.9) is installed on your system.
+- Install the [Python Plugin for IntelliJ](https://plugins.jetbrains.com/plugin/631-python).
+- Enable Python support in IntelliJ:
+  - Go to `File > Project Structure > Facets > Add - Python`.
+  - Add a Python environment by configuring the Python interpreter. 
+  ![IntelliJ Python Facet Configuration](./images/facets-config.png)
+  - Add a module in IntelliJ by navigating to `File > Project Structure > Modules > Add - Python`.
+  ![IntelliJ Module Configuration](./images/module-config.png)
```

Comment:
Lets fix the formatting here
```suggestion
  - Add a Python environment by configuring the Python interpreter.
  
     ![IntelliJ Python Facet Configuration](./images/facets-config.png)
  - Add a module in IntelliJ by navigating to `File > Project Structure > Modules > Add - Python`.

    ![IntelliJ Module Configuration](./images/module-config.png)
```

Reply:
Could you also please update the screenshots? 
For me it looks different by now - how does your configuration look like? Is this only due to some local changes on my setup?

![image](https://github.com/user-attachments/assets/2e82e256-8d70-43a9-9d88-f993f1d55f21)
![image](https://github.com/user-attachments/assets/d6313adf-75c7-4ccf-b676-147a532fd9f7)

Reply:
Without the empty line and the spacing at the start the pictures are not aligned properly for me 
![image](https://github.com/user-attachments/assets/b7f7a971-1156-464a-b846-e28cf53de64b)